indico/modules/rb_new/client/js/reducers/roomBooking/roomList.js

```diff
@@ -28,11 +28,11 @@ const initialRoomsState = {
 
 
 function mergeRooms(state, action) {
-    const {list: oldList} = state;
-    const {loadMore, matching, total, rooms: newList} = action;
+    const {list: oldList, total: stateTotal} = state;
+    const {loadMore, matching, rooms: newList, total} = action;
     return {
         matching,
-        total,
+        total: total === null ? stateTotal : total,
```

Comment:
Is there a reason why you comapre to `null` instead of just `total ? total : stateTotal`

Reply:
`total === 0` in case on on results

Reply:
yes, `total ?  total : stateTotal` will return `stateTotal` for `total = 0` which I don't want. This whole change is based on the fact that we calculate `total` on the backend only on the first request. After that we don't return `total` thus `null`.

Reply:
Okay :) thanks